.prettierignore

```diff
@@ -0,0 +1,2 @@
+# This is a machine-generated and read file, no need to prettify it
```

Comment:
I would normally agree with you, but since the goal here is to always use this script to update the Pipfile.lock (because otherwise tests fail) then I think we don't need to run it through prettier, especially since the raw output is not badly formatted, it just uses 4-space indents but prettier is configured to use 2-space indents (Do you know why it was set up like this, by the way? ggshield contains no other json files)

Regarding IDE: I assume they don't edit the file themselves, they just run `pipenv install` without `--skip-lock`, which updates the Pipfile.lock (and breaks Python 3.7 installs, unless it was ran using Python 3.7).

But if you think the file should really be 2-space indented then I will make the script itself re-indent the file.

Reply:
ok we can try.